pages/deployment/wifi.md

```diff
@@ -1,6 +1,60 @@
+* [Captive Portal Network Setup](/pages/deployment/wifi.md#captive-portal-network-setup)
+
+## Introduction
+
+In order to deploy code to your device(s), they will need to be connected to the internet in some shape or form. The resin.io HostOS manages your devices' network connection with [ConnMan][connman], an open source connection manager.
+
+Currently all resin.io devices are configured to favour [ethernet](/pages/deployment/wifi.md#ethernet-connections) and will automatically use this connection type if it is available. You can define all manner of network configurations using [Connman][connman], for example [setting a static IP](/pages/deployment/wifi.md#set-static-ip).
+
+## Network Requirements
+
+In order for a resin.io device to get outside of the local network and connect to the resin.io API, there are a few core network requirements.
+
+Resin.io makes use of the following ports:
+
+* `443` TCP - This is the most fundamental requirement - it is used to connect to the VPN and the web terminal, and many web endpoints using TLS (https://.)
+* `123` UDP - For NTP time synchronisation.
+* `53` UDP - For DNS name resolution.
+
+Each of these should work with outward only (and inward once outward connection established) firewall settings.
+
+Additionally, if the network your device is connecting to works with whitelisting, you should whitelist the following domains on Port `80` and `443`:
+* `*.resin.io`
+* `*.pubnub.com`
+* `*.ngrok.com`
+* `*.ngrok.io`
+
+## Ethernet Connections
+
+As mentioned earlier, resin.io HostOS defaults to using ethernet for internet connectivity, so if you connect an active ethernet cable to your device and all the [network requirements](/pages/deployment/wifi.md#network-requirements) are satisfied, your device should simply just connect to dashboard.resin.io and you should be able to push code to it.
+
+If this this is not the case, and your device is still not online 10 minutes after power up, then give us a shout at `support@resin.io` or click on the small **`?`** in the bottom right of your resin.io dashboard.
+
+### Set Static IP
+
+In order to configure static IP on a pre-provisioned SD card perform the following steps:-
+* Mount the FAT partitions of the OS image either directly from the `.img` file, or by burning the SD card and mounting it on your computer. The volume will be called `resin-conf`.
+* Inside `resin-conf` you will find a `config.json` file and in it you will see a network key/value pair which contains JSON string encoded [Connman][connman] settings. Now edit the value to include the following entry, replacing `<static IP>` with your desired static IP:-
+
+```
+[service_home_ethernet]
+Type = ethernet
+IPv4 = <static IP>/255.255.255.0/192.168.1.1
+Nameservers = 192.168.1.1,8.8.8.8
+```
+Note that this assumes your network gateway is 192.168.1.1, this can vary so adjust this according to your local network configuration.
```

Comment:
missed seeing this before the above comment - still might be clearer if we mention it as the gateway above?